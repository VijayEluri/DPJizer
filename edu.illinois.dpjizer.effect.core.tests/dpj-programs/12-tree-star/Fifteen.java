/**
 * This file is licensed under the University of Illinois/NCSA Open Source License. See LICENSE.TXT for details.
 */

/**
 * 
 * @author Mohsen Vakilian
 * 
 */
class C<region R1> {
	int []<R1:[i]>#i a in R1;

	public C() {
	}

	void m() {
		if (a[0] == 0)
			new C<R1:[0]>().m();
	}
}