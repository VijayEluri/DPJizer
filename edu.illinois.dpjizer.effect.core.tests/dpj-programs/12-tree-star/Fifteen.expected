C.C() invokes java.lang.Object.Object()
C.m() reads R1, R1 : [0] invokes C.m() where { this -> R1 : [0] : *, R1 -> R1 : [0] }
